cmake/TenzirRegisterPlugin.cmake

```diff
@@ -245,13 +263,14 @@ function (TenzirCompileFlatBuffers)
     # FBS_TARGET cannot depend on the OUTPUT of a add_custom_command directly,
     # and depending on a BYPRODUCT of a add_custom_command causes the Unix
     # Makefiles generator to constantly rebuild targets in the same export set.
-    add_custom_target("compile-flatbuffers-schema-${basename}"
+    add_custom_target("compile-flatbuffers-schema-${FBS_TARGET}-${basename}"
                       DEPENDS "${desired_file}")
-    add_dependencies(${FBS_TARGET} "compile-flatbuffers-schema-${basename}")
+    add_dependencies(${FBS_TARGET}
+                     "compile-flatbuffers-schema-${FBS_TARGET}-${basename}")
     set_property(
       TARGET ${FBS_TARGET}
       APPEND
-      PROPERTY PUBLIC_HEADER "${desired_file}")
+      PROPERTY PUBLIC_HEADER "${desired_file}" "${schema}")
```

Comment:
This is a bit of a hack to install the `*.fbs` files—we could also just install the entire directory into a location other than `<includedir>/tenzir/include/fbs` with a bit more effort, if that's preferred.